client/src/utils.rs

```diff
@@ -60,24 +62,28 @@ pub fn get_councillors(api: &Api) -> Result<Vec<AccountId>> {
 		.ok_or_else(|| ApiClientError::Other("Couldn't get councillors".into()))
 }
 
-pub fn send_and_wait_for_in_block<C: Encode>(
+pub fn send_and_wait_for_in_block<C: Encode + Clone>(
 	api: &Api,
 	xt: EncointerXt<C>,
 	tx_payment_cid: Option<&str>,
 ) -> Option<H256> {
-	send_xt_hex_and_wait_for_in_block(api, xt.hex_encode(), tx_payment_cid)
+	send_xt_hex_and_wait_for_in_block(api, xt, tx_payment_cid)
 }
 
-pub fn send_xt_hex_and_wait_for_in_block(
+pub fn send_xt_hex_and_wait_for_in_block<C>(
```

Comment:
If you still wish to have a wrapper function such as that (without a known xt type), instead of `hex` you could use the `opaque` version: `submit_and_watch_opaque_extrinsic_until`. This takes `Bytes` instead of a `String`.